coderadar-ui/src/app/view/configure-project/configure-project.component.ts

```diff
@@ -17,6 +17,23 @@ export interface DialogData {
   displayNames: string[];
 }
 
+@Component({
+  selector: 'app-dialog-overview-example-dialog',
+  templateUrl: 'app-merge-dialog.html',
+})
+export class MergeDialogComponent {
```

Comment:
Please create one file for each class or interface

Reply:
Done